syftbox/server/emails/router.py

```diff
@@ -17,21 +17,20 @@ async def send_email(
     email_request: SendEmailRequest,
     server_settings: ServerSettings = Depends(get_server_settings),
 ) -> bool:
-    if not server_settings.email_service_api_key:
+    if not server_settings.sendgrid_secret:
         raise httpx.HTTPStatusError("Email service API key is not set", request=None, response=None)
```

Comment:
Suggestion: we can use `fastapi.HTTPException` for any endpoint exceptions, anything else probably gets transformed into a 50x by starlette